docs/vgclog_examples.md

```diff
@@ -0,0 +1,594 @@
+
+```xml
+<af-start id="3" threadId="0000000000AC4F80" totalBytesRequested="272" timestamp="2020-10-18T13:27:09.603" intervalms="1913.921" type="nursery" />
+```
+
+The `<cycle-start>` element marks the start of the cycle. The attribute `type="scavenge"` confirms that this activity is a *scavenge* partial GC cycle:
+
+```xml
+<cycle-start id="4" type="scavenge" contextid="0" timestamp="2020-10-18T13:27:09.603" intervalms="1913.959" />
+```
+
+Most elements are labeled with an `id` attribute that increases in value incrementally, a`timestamp` attribute, and a `contextid` attribute. All elements that record GC increments and operations that are associated with a particular cycle have a `contextid` value that matches the `id` value of the cycle. The `<cycle-start>` element of this example cycle has a `id="4"`, so all subsequent elements that have a `contextid="4"`, such as the `<gc-start>` increment element and the `<gc-op>` operation element, are associated with this particular example cycle.  
+
+The `<gc-start>` element records the first GC increment. In this section, you can find information about the amount of memory available (`<mem-info>`) and where it is located in the heap.
+
+The memory snapshot within the `<gc-start>` element is taken before the *scavenge* operation and can be compared with a similar snapshot that is taken afterwards to understand the effect on the heap.
+
+```xml
+<gc-start id="5" type="scavenge" contextid="4" timestamp="2020-10-18T13:27:09.603">
+
+<mem-info id="6" free="802568640" total="1073741824" percent="74">
+
+<mem type="nursery" free="0" total="268435456" percent="0">
+
+<mem type="allocate" free="0" total="134217728" percent="0" />
+
+<mem type="survivor" free="0" total="134217728" percent="0" />
+
+</mem>
+
+<mem type="tenure" free="802568640" total="805306368" percent="99">
+
+<mem type="soa" free="762302912" total="765040640" percent="99" />
+
+<mem type="loa" free="40265728" total="40265728" percent="100" />
+
+</mem>
+
+<remembered-set count="30758" />
+
+</mem-info>
+
+</gc-start>
+
+<allocation-stats totalBytes="136862352" >
+
+<allocated-bytes non-tlh="7097008" tlh="129765344" />
+
+<largest-consumer threadName="Start Level: Equinox Container: c25c3c67-836a-49e7-9949-a16d2c2a5a4d" threadId="000000000045E300" bytes="46137952" />
+
+</allocation-stats>
+```
+
+In this example, the *nursery* area has no free space available in either the *allocate* or *survivor* spaces. The *tenure* area has 99% free in the small object area (SOA). The large object area (LOA) contains no objects.
+
+The *scavenge* GC operation starts with the `<gc-op>` element; child elements record details about the operation. For example, `<scavenger-info>` shows that the *tenure age* is set to `1`. For a *tenure age* of `1`, any objects that are copied between the *allocate* and *survivor* space once are moved to the *tenure* area during the next *scavenge* operation.
```

Comment:
Last sentence sounds somewhat awkward or I don't understand it.

Should be: any object in allocate space with age below or equal tenure age will be moved/copied to survivor, and above tenure age will be moved to tenure area.

Reply:
**Is this correct?**

The `<memory-copied>` element indicates that 6 MB (6027440) of reachable objects were moved by the *scavenge* operation from the allocate space to the survivor space in the nursery area, and 562KB(562848) were moved to the tenure area. Anything above tenure age will be moved to the tenure area. For more information about how the scavenge operation acts on the heap, see [`gencon` policy(default)](gc.md#gencon-policy-default).

The `<scavenger-info>` element shows that the *tenure age* is set to `7` so any object in the *allocate* space with age below or equal to a tenure age of `7` is copied to the survivor space during the `scavenge`operation.